tests/mol-tts_single/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title>Single text-to-speech fallback for MO</title>
+</head>
+<body>
+    <p>Test passes (i.e., the Reading System correctly falls back on Text-to-Speech when no audio file is present) if</p>
+    <ol>
+        <li>The Reading System provides the user with a way to turn text reading on</li>
+        <li>If turned on, the extract on the next page is read aloud by the Reading System using Text-to-Speech</li>
+        <li>If the Reading System implements the association with style information, the text being read is highlighted with a green background and the rest of the text is grayed out</li>
```

Comment:
Same question about whether this is a true requirement. Either the "if" should change, or this should be clearly marked as optional.

Reply:
And again, maybe something like "If the Reading System implements styling for media overlays, ..."

Reply:
Same answer :-)